roles/rds-apb-openshift/tasks/main.yml

```diff
@@ -36,9 +38,10 @@
     ports:
     - name: "{{ db_engine }}"
       protocol: "TCP"
-      port: "{{ port }}"
-      target_port: "{{ port }}"
+      port: 5432
```

Comment:
I don't understand it wouldn't work as it was before. Could you explain?

Reply:
It is not working in Kubernetes with target_port: "{{ port }}" as Kubernetes does not accept port numbers as strings.